### DISCUSSION TITLE
Can this addon import .rdc files not acquired from Maps or Chrome?

### DISCUSSION THREAD

**Discussion by ehlektronix** [eliemichel/MapsModelsImporter#178] [Q&A]
I've been trying to import a .rdc file to Blender that comes from a game, but whenever I try to import it, nothing gets added to my Blender scene. I tried again with one of the sample files you've provided, and it works fine. I'm an absolute noob when it comes to coding and stuff, but is there a way to make this work?

**Comment by eliemichel** [comment]
The RDC capture must have been done on google maps (or earth web), it does not work for other sources! Two reasons: (a) it is more complicated to make it work for any program and more importantly (b) I don't want this tool to be used to steal artistic material.
